stripe/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -326,7 +327,27 @@ data class ConfirmPaymentIntentParams internal constructor(
         fun create(
             clientSecret: String,
             returnUrl: String? = null,
-            extraParams: Map<String, Any>? = null,
+            shipping: Shipping? = null
+        ): ConfirmPaymentIntentParams {
+            return ConfirmPaymentIntentParams(
+                clientSecret = clientSecret,
+                returnUrl = returnUrl,
+                shipping = shipping
+            )
+        }
+
+        /**
+         * Create a [ConfirmPaymentIntentParams] without a payment method.
+         */
+        @JvmOverloads
+        @JvmStatic
+        @Deprecated(
+            "The [extraParams] property is deprecated and will be removed in an upcoming major version. Use the [setupFutureUsage] to specify a value for setup_future_usage."
```

Comment:
[setupFutureUsage] is not exposed in this method, we should probably point to [createWithPaymentMethodId] here?

Reply:
Ah good point. I'll add a setupFutureUsage parameter to the non-deprecated version.